CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
-## Unreleased
+## v1.3.0-rc.0 / 2018-03-23
 
-* [FEATURE] Add `kube_pod_completion_time` metric.
```

Comment:
Another important feature:
* add secret and configmap metrics. #379

Reply:
Thanks! Added.